sconscontrib/SCons/Tool/noweb/SConstruct

```diff
@@ -0,0 +1,10 @@
+import os
+
+env = Environment()
+
+db_site = os.path.join(os.path.expanduser('~'),'.scons','site_scons','site_tools','noweb')
```

Comment:
forcing install in ```~/.scons``` ?

Reply:
There is no "forcing" here, the user has to explicitly call "scons install" (it's just a convenience feature). But maybe I'm not getting what you're actually aiming at here, sorry. Can you say what you would **rather** see?

Reply:
Ahh.. sorry I misread it.
Maybe add a note in the README.rst that is where it will install via SConstruct?

Reply:
Okay, done.